compiler/src/main/java/io/neow3j/compiler/converters/ArraysConverter.java

```diff
@@ -41,7 +50,8 @@ public AbstractInsnNode convert(AbstractInsnNode insn, NeoMethod neoMethod,
                 if (isByteArrayInstantiation(insn)) {
                     insn = handleNewByteArray(insn, neoMethod);
                 } else {
-                    neoMethod.addInstruction(new NeoInstruction(OpCode.NEWARRAY));
+                    handleNewArray(insn, neoMethod);
+                    // TODO!!!
```

Comment:
What do you mean by this `TODO` here? Can you explain a bit more in detail?

Reply:
Removed it.